helpers/errorCodes.js

```diff
@@ -54,6 +54,14 @@ module.exports = {
 		code: 'E0012',
 		message: 'Exercise not found'
 	},
+  E0010: {
+    code: 'E0010',
+    message: 'Email could not be sent'
+  },
+  E0011: {
+    code: 'E0011',
+    message: 'Content creator not found'
+  },
```

Comment:
Change errorCodes based on MARKDOWN file find in resources. So E0011 should be E0013.

When adding errorCodes, first consult the ERRORS.md, and add accordingly --> Dont add over existing errorcodes. 

Ofcourse this might be a timing issue where you didn't check it right after someone had added some errors.

Reply:
yes it was made before the new changes, I will fix it